### PR TITLE
Formatted markedown message3

### PR DESCRIPTION
**Pull request by jopaleti**
#### Ticket Link

```release-note
NONE
```
-->

### CODE REVIEW

**Comment by M-ZubairAhmed** [comment]
@jopaleti please link the Github issue and Jira ticket in the description

File: server/.golangci.yml
Comment (M-ZubairAhmed) [inline server/.golangci.yml:1]: please revert not related change

File: server/channels/api4/ip_filtering_test.go
Comment (M-ZubairAhmed) [inline server/channels/api4/ip_filtering_test.go:1]: please revert not related change

File: webapp/package-lock.json
Comment (M-ZubairAhmed) [inline webapp/package-lock.json:1]: please revert not related change

File: webapp/package.json
Comment (M-ZubairAhmed) [inline webapp/package.json:1]: please revert not related change

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

**Comment by jopaleti** [comment]
@M-ZubairAhmed I have submitted a new PR on [https://github.com/mattermost/mattermost/pull/28657](https://github.com/mattermost/mattermost/pull/28657)

**Comment by jopaleti** [comment]
@M-ZubairAhmed I have submitted a new PR on [https://github.com/mattermost/mattermost/pull/28657](https://github.com/mattermost/mattermost/pull/28657)

**Pull request closed without merging**